angular-ngc/libraries/lib-a/src/test.ts

```diff
@@ -0,0 +1,13 @@
+import 'zone.js';
```

Comment:
I want to avoid having to copy and paste this file every single time. I think it should live in `tools/testing/bootstrap.ts` or something like that so it can be compiled once on its own and then the test macro includes it in the bundle... Then it gets loaded as a bootstrap script *before* the spec files...

Reply:
I created a `karma.polyfills.ts` to contains the `import`, I will move it to `tools/testing/bootstrap.ts` later.

Reply:
I removed this file and use `write_file` to generate it.